src/notebooks/phdi_analytics.py

```diff
@@ -0,0 +1,297 @@
+import csv
+import datetime as dt
+import os
+import random
+import re
+
+import censusgeocode as cg
+from concurrent.futures import ThreadPoolExecutor
+from faker import Faker
+from geopy import Point
+from geopy.distance import geodesic
+import geopandas as gpd
+import git
+import matplotlib.ticker as mtick
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import names
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from tqdm.notebook import tqdm
+
+
+def get_git_root(path):
```

Comment:
This seems like overkill for a prototype notebook that will only ever reference one Git repo. I'd recommend removing this function, removing the `git` module dependency, and simply hardcoding the path.